client/components/post-stats-card/stories/index.stories.jsx

```diff
@@ -1,6 +1,6 @@
 import PostStatsCard from '../';
 
-export default { title: 'packages/components/Post Stats Card' };
+export default { title: 'client/components/Post Stats Card' };
```

Comment:
This is follow-up territory, but we should uphold good Storybook standards for these client components to set a good example. When making new stories, people (or AI) may copy what is here.

Make sure they're leveraging TS-based controls, simple CSF3, no meaningless stories, no showcasing of bad practices, etc.